src/nanoemoji/write_glyphmap_for_glyph_svgs.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates a glyphmap for svgs named by glyph id."""
+from absl import app
+from absl import flags
+from absl import logging
+from nanoemoji.glyphmap import GlyphMapping
+from nanoemoji import util
+from pathlib import Path
+
+
+FLAGS = flags.FLAGS
+
+
+flags.DEFINE_string(
+    "log_level",
```

Comment:
I don't see any logging being used for this particular entry point

Reply:
Oops, fixed.